tests/test_github.py

```diff
@@ -975,6 +1020,16 @@ def test_fetch_pulls_until_date(self):
                                    'Link': '<' + GITHUB_ISSUES_URL + '/?&page=2>; rel="next", <' +
                                            GITHUB_ISSUES_URL + '/?&page=3>; rel="last"'
                                })
+
+        httpretty.register_uri(httpretty.GET,
```

Comment:
In the test `test_fetch_pulls_until_date` it seems that no assertion is done over the commit comments data

Reply:
Sure, will add it ;)